pallets/automation-time/src/tests.rs

```diff
@@ -1,7 +1,5 @@
-// use super::*;
-// use mock::*;
 use crate::{mock::*, Error, Task};
-use frame_support::{assert_noop, assert_ok};
+use frame_support::{assert_noop, assert_ok, traits::OnInitialize};
```

Comment:
Would be nice to see a test for the edge case where we overload the task list (ie we're at capacity).

Reply:
I will add that to the overflow tests. Right now the code stops that from happening by WAY under allocating how many tasks it can do per block